src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -470,6 +470,7 @@ def forward(
         self,
         input_ids: torch.Tensor,
         attention_mask: Optional[torch.Tensor] = None,
+        inputs_embeds: torch.Tensor = None,
```

Comment:
```suggestion
        attention_mask: Optional[torch.Tensor] = None,
        inputs_embeds: Optional[torch.Tensor] = None,
```
As I believe the input embedings are optionally passed down here